src/core/store.ts

```diff
@@ -48,10 +93,31 @@ type StateListener = () => void
 type MountedAtoms = Set<AnyAtom>
 
 // store methods
+/**
+ * Read an atom's hidden [AtomState]. Derived atom state may be recomputed if they
+ * are invalidated and any of their transitive dependencies have changed.
+ */
 export const READ_ATOM = 'r'
+/**
+ * Write an atom's Value.
+ */
 export const WRITE_ATOM = 'w'
+/**
+ * Commit pending writes to an atom.
+ */
 export const COMMIT_ATOM = 'c'
+/**
+ * Add a subscriber function to an atom. The atom will become "mounted" if this
```

Comment:
To be precise, an atom will be mounted if a) it has a subscriber, or b) another atom depend on it.

Reply:
@justjake Just to make sure if you are aware that a mounted atom may not have any listeners, are you?

Reply:
Yes. I think I fixed the comment.

Reply:
good. will you fix here or any reason not?

Reply:
Yes!